src/stores/my-account.ts

```diff
@@ -67,10 +122,24 @@ export const useMyAccount = create<State & Actions>()((set, get) => ({
       const signer = await loginWithSecretKey(secretKey)
       const encodedSecretKey = encodeSecretKey(secretKey)
       address = signer.address
+      const randomPicture = createAvatar(bottts, {
+        size: 128,
+        seed: address,
+      }).toDataUriSync()
+      const name = generateRandomName(address)
+      const authenticatedUser = {
+        profilePic: randomPicture,
+        address,
+        encodedSecretKey,
+        name,
+      } as AuthenticatedUser
       set({
         address,
         signer,
         encodedSecretKey,
+        authenticatedUser,
+        isAuthenticated: true,
+        authMethod: AUTHENTICATION_METHODS.Anonymous,
```

Comment:
so I think you only need this line change to add authMethod to the loginAnonymously.
other changes are not really needed